diff of execution of:
echo 'cat < infile > out1 < infile > out2 > out3; ls -l --time-style=+' | tr ';' '\n' | [shell]
--- expected	2024-08-27 17:55:05.892580108 +0200
+++ found	2024-08-27 17:55:05.912580411 +0200
@@ -1,11 +1,11 @@
 total 8
 -rw-rw-r-- 1 bruno bruno 34  infile
--rw-rw-r-- 1 bruno bruno  0  out1
--rw-rw-r-- 1 bruno bruno  0  out2
--rw-rw-r-- 1 bruno bruno 34  out3
+-rw-r--r-- 1 bruno bruno  0  out1
+-rw-r--r-- 1 bruno bruno  0  out2
+-rw-r--r-- 1 bruno bruno 34  out3
 -r--r--r-- 1 bruno bruno  0  readonly
 
 ======== stderr =======
 
 ======== return =======
 return value: 0
